jupyter-extension/src/plugins/mount/components/Pipeline/Pipeline.tsx

```diff
@@ -169,8 +187,8 @@ const Pipeline: React.FC<PipelineProps> = ({
           data-testid="Pipeline__specPreview"
           name="specPreview"
           value={`pipeline:
-  name: ${pipeline.name}
-  project: ${pipeline.project?.name ?? 'default'}
+  name: ${pipelineName}
+  project: ${pipelineProject ?? 'default'}
```

Comment:
I don't think this is working. When I have no project specified, this field is empty in the pipeline spec preview

Reply:
good catch, patched this just now